File: 1165.py
for l in range(0, int(input())):
    n = int(input())
    primo = 0
    for i in range(1, n+1):
        if n % i == 0:
            primo+=1
    if primo == 2 and 1 != n:
        print(f'{n} eh primo')
    else:
        print(f'{n} nao eh primo')

File: 1165.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let a = Number(lines.shift());
for(var l = 0; l < a; l++){
    let n = Number(lines.shift());
    let primo = 0;
    for(var i = 1; i < n+1; i++){
        if(n % i === 0){
            primo += 1;
        }
    }
    primo === 2 && 1 !== n ? console.log(`${n} eh primo`): console.log(`${n} nao eh primo`);
}
